source/MaterialXView/ShaderValidators/Glsl/GlslValidator.h

```diff
@@ -31,21 +31,29 @@ class GlslValidator
 {
   public:
     /// Structure to hold information about program inputs
+    /// The structure is populated by directly scanning the program so may not contain
+    /// some inputs listed on any associated HwShader as those inputs may have been
+    /// optimized out if they are unused.
     struct ProgramInput
     {
         static int INVALID_OPENGL_TYPE;
 
         /// Program location. -1 means an invalid location
         int _location;
         /// OpenGL type of the input. -1 means an invalid type
-        int _type;
+        int _gltype;
```

Comment:
Code convention: not using _ prefix on public members

Reply:
Didn't know. I'll fix.